src/components/SideNavBar/SideNavBar.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component, PropTypes } from 'react';
+import { Link } from 'react-router';
+import '../../styles/Common.scss';
+import LeftPanel from '../../containers/LeftPanel';
+import Header from '../../containers/Header';
+import * as constants from '../../constants';
+
+const SideNavBar = (props) => {
+  const { toggleShowItem, showItem } = props;
```

Comment:
you can avoid this line, using the arguments in the function:
` const SideNavBar = ({ toggleShowItem, showItem }) => {`